src/jit/lsra.cpp

```diff
@@ -10929,11 +10945,58 @@ void LinearScan::resolveEdge(BasicBlock*      fromBlock,
                 else
                 {
                     compiler->codeGen->regSet.rsSetRegsModified(genRegMask(tempReg) DEBUGARG(dumpTerse));
-                    assert(sourceIntervals[targetReg] != nullptr);
```

Comment:
This should be moved to the else clause instead of deleted.

Reply:
Right. Thanks.